pandas/tseries/converter.py

```diff
@@ -222,31 +220,6 @@ def __init__(self, locator, tz=None, defaultfmt='%Y-%m-%d'):
         # matplotlib.dates._UTC has no _utcoffset called by pandas
         if self._tz is dates.UTC:
             self._tz._utcoffset = self._tz.utcoffset(None)
-        self.scaled = {
-            365.0: '%Y',
-            30.: '%b %Y',
-            1.0: '%b %d %Y',
-            1. / 24.: '%H:%M:%S',
-            1. / 24. / 3600. / 1000.: '%H:%M:%S.%f'
```

Comment:
Found the difference between what was here and what is in MPL it is the extra 1000 in the last entry.

I will have a PR shortly.